City-of-Helsinki/tilavarauspalvelu-core
# Migrate to python 3.12

## 🛠️ Changelog
- Migrates the project to Python 3.12

## 🧪 Test plan

- Create a new virtual environment locally for python 3.12:
    - `poetry env use "<path to python 3.12 executable"` -> `poetry install`
- Run tests with new environment
- Run Django admin and graphQL with new environment

CI already runs tests and builds image, so those should be tested with that
